srcpkgs/CPU-X/template

```diff
@@ -1,25 +1,27 @@
 # Template file for 'CPU-X'
 pkgname=CPU-X
-version=4.3.0
-revision=2
+version=4.5.2
+revision=1
 archs="x86_64* i686*"
 build_style=cmake
 hostmakedepends="pkg-config nasm"
 makedepends="ncurses-devel gettext-devel libcpuid-devel pciutils-devel
- glfw-devel ocl-icd-devel procps-ng-devel libstatgrab
- $(vopt_if gtk3 gtk+3-devel)"
+ glfw-devel ocl-icd-devel procps-ng-devel libstatgrab gtk+3-devel
+ vulkan-loader Vulkan-Headers"
+checkdepends="mawk nawk grep"
 short_desc="Free software that gathers information on CPU, motherboard and more"
-maintainer="Subhaditya Nath <sn03.general@gmail.com>"
+maintainer="Orphan <orphan@voidlinux.org>"
 license="GPL-3.0-or-later"
-homepage="https://x0rg.github.io/CPU-X/"
-changelog="https://raw.githubusercontent.com/X0rg/CPU-X/master/ChangeLog.md"
-distfiles="https://github.com/X0rg/CPU-X/archive/v${version}.tar.gz"
-checksum=6ad7a8ac2d6c687a38a895fbbfbf2de690311676ac374d5857588bb983839433
-build_options="gtk3"
-build_options_default="gtk3"
-# needs grep -P which is not available in chroot-grep and replacing it with
-# grep breaks base-chroot
-make_check=no
+homepage="https://thetumultuousunicornofdarkness.github.io/CPU-X/"
+changelog="https://github.com/TheTumultuousUnicornOfDarkness/CPU-X/blob/master/ChangeLog.md"
+distfiles="https://github.com/thetumultuousunicornofdarkness/CPU-X/archive/v${version}.tar.gz"
+checksum=6cdab38928ca5684b22f7b8d868d42a25a48be6da813743b6b59a88d6e9f25d1
+
+if [ "$XBPS_BUILD_ENVIRONMENT" != void-packages-ci ]; then
+	# Installing the packages in checkdepends causes base-chroot to break.
+	# So, only run the checks in CI, where the chroot is ethereal anyway.
+	make_check=no
+fi
```

Comment:
grep specifically breaks base-chroot because base-chroot depends on chroot-grep. Is chroot-grep not enough for tests?

Reply:
Nope. chroot-grep is built without perl regexp support, and the tests use them. So, when the tests call `grep -P`, grep fails with an error message saying that grep has been built without perl regexp support, and since grep fails, the tests also fail.